README.md

```diff
@@ -306,9 +311,66 @@ class ReportingOperation {
 register('send-report', ReportingOperation);
 ```
 
-## Keep-alive worklet
+### Keeping a worklet alive for multiple operations
+
+Callers may wish to run multiple worklet operations from the same context, e.g. they might select a URL and then send one or more aggregatable reports. To do so, they would need to use the `keepAlive: true` option when calling each of the worklet operations (except perhaps in the last call, if there was no need to extend the worklet's lifetime beyond that call).
+
+As an example, in the embedder page:
+
+```js
+// Load the worklet module.
+await window.sharedStorage.worklet.addModule('worklet.js');
+
+// Select a URL, keeping the worklet alive.
+const fencedFrameConfig = await window.selectURL(
+  [
+    {url: "blob:https://a.example/123…"},
+    {url: "blob:https://b.example/abc…"}
+  ],
+  {
+    data: { ... },
+    keepAlive: true,
+    resolveToConfig: true
+  }
+);
+
+// Navigate a fenced frame to the resulting config.
+document.getElementById('my-fenced-frame').config = fencedFrameConfig;
+
+// Send some report, keeping the worklet alive.
+await window.sharedStorage.run('report', {
+  data: { ... },
+  keepAlive: true,
+});
+
+// Send another report, allowing the worklet to close afterwards.
+await window.sharedStorage.run('report', {
+  data: { ... },
+});
+
+// From this point on, if we make any additional worklet calls, they will fail.
+```
+
+In the worklet script (`worklet.js`):
+
+```js
+class URLOperation {
+  // See previous examples for how to write a `selectURL()` operation class.
+  async run(urls, data) { ... }
+}
+
+class ReportOperation {
+  // See previous examples for how to write a `run()` operation class.
+  async run(data) { ... }
+}
+
+register('select-url', URLOperation);
+register('report', ReportOperation);
+```
+
+## Worklets can outlive renderers
```

Comment:
I think it’s more natural to say that "Worklets can outlive the associated Document", as I’m not sure if "renderer" is a standard term.